deployments/helm-chart/README.md

```diff
@@ -184,6 +184,7 @@ Parameter | Description | Default
 `controller.service.type` | The type of service to create for the Ingress controller. | LoadBalancer
 `controller.service.externalTrafficPolicy` | The externalTrafficPolicy of the service. The value Local preserves the client source IP. | Local
 `controller.service.annotations` | The annotations of the Ingress controller service. | {}
+`controller.service.extraLabels` | The extra labels of the service. | {}
```

Comment:
Are these extra labels that are already in use by an existing service? Or extra labels to add to a Service that we're deploying?

Reply:
@jputrino extra labels to add to a Service that we're deploying

Reply:
```suggestion
`controller.service.extraLabels` | Defines extra labels to add to the service. | {}
```